src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -172,6 +174,28 @@ class BundleSpec extends ChiselFlatSpec with BundleSpecUtils with Utils {
     }
   }
 
+  "Calling litValue on Bundle containing DontCare" should "make a decent error message" in {
+    class MyBundle extends Bundle {
+      val a = UInt(8.W)
+      val b = Bool()
+      val c = UInt(4.W)
+    }
+
+    class Example extends RawModule {
+      val out = IO(Output(new MyBundle))
+      val lit = (new MyBundle).Lit(_.a -> 8.U, _.b -> true.B)
+      out := lit
+      println(lit.litValue)
+    }
+
+    val x = intercept[ChiselException] {
+      (new ChiselStage).execute(Array("--throw-on-first-error"), Seq(ChiselGeneratorAnnotation(() => new Example)))
```

Comment:
```suggestion
      ChiselStage.elaborate(new Example , Array("--throw-on-first-error"))
```
I prefer the object methods and avoiding annotations where possible.